src/utility.js

```diff
@@ -165,8 +167,10 @@ function getTooltipContent(element) {
  * @return {number} Value with the collision flags.
  */
 function getViewportCollisions(coords, elementWidth, elementHeight) {
-	var viewportTop = session.scrollTop,
-		viewportLeft =  session.scrollLeft,
+	// adjusting viewport even though it might be negative because coords
+	// comparing with are relative to compensated position
+	var viewportTop = session.scrollTop - session.positionCompensation.top,
+		viewportLeft =	session.scrollLeft - session.positionCompensation.left,
```

Comment:
These seem to be causing the collision detection to break when the position is relative. Will explain the bug in PR comments.

Are you sure this is the correct place to apply the adjustment? These variables (`viewportTop`, `viewportLeft`) are the viewport position relative to the page.

![desktop_page](https://cloud.githubusercontent.com/assets/141884/22629536/d46c1e76-eb9c-11e6-859b-c6c5e960120b.jpg)
_(Source: http://www.quirksmode.org/mobile/viewports.html)_


Since the `positionCompensation` adjustments are based on the CSS, I think the adjustments need to be applied in the maths below, which compare the `CSSCoordinates` values to the viewport position to determine if there is a collision.